libpod/networking_linux.go

```diff
@@ -50,7 +50,14 @@ func (r *Runtime) getPodNetwork(id, name, nsPath string, networks []string, port
 
 // Create and configure a new network namespace for a container
 func (r *Runtime) configureNetNS(ctr *Container, ctrNS ns.NetNS) ([]*cnitypes.Result, error) {
-	podNetwork := r.getPodNetwork(ctr.ID(), ctr.Name(), ctrNS.Path(), ctr.config.Networks, ctr.config.PortMappings, ctr.config.StaticIP)
+	var requestedIP net.IP
+	if ctr.requestedIP != nil {
+		requestedIP = ctr.requestedIP
```

Comment:
I think we probably want to set ctr.requestedIP to nil after we do this assignment, in case we reuse the container later and don't necessarily want the same IP

Reply:
Good point. I was also thinking about it but thought it might not be necessary. Will add it and force push.